scrapy/mail.py

```diff
@@ -4,11 +4,6 @@
 See documentation in docs/topics/email.rst
 """
 import logging
-
-try:
-    from cStringIO import StringIO as BytesIO
-except ImportError:
-    from io import BytesIO
 import six
```

Comment:
Considering we are using `six`, I'd go for https://six.readthedocs.io/#six.BytesIO

Reply:
Your suggestions seem better, I have made the changes accordingly.Thank you.